fhir-server-test/src/test/java/com/ibm/fhir/server/test/FHIRServerTestBase.java

```diff
@@ -1,763 +1,779 @@
-        }
-        else {
-            assertTrue("Passed with no issues in validation", true);
-        }
-    }
-
-    /**
-     * Parses a location URI into the resourceType, resourceId, and (optionally) the version id.
-     * @param location
-     * @return
-     */
-    public static String[] parseLocationURI(String location) {
-        String[] result = null;
-        if (location == null) {
-            throw new NullPointerException("The 'location' parameter was specified as null.");
-        }
-
-        String[] tokens = location.split("/");
-        // Check if we should expect 4 tokens or only 2.
-        if (location.contains("_history")) {
-            if (tokens.length >= 4) {
-                result = new String[3];
-                result[0] = tokens[tokens.length - 4];
-                result[1] = tokens[tokens.length - 3];
-                result[2] = tokens[tokens.length - 1];
-            } else {
-                throw new IllegalArgumentException("Incorrect location value specified: " + location);
-            }
-        } else {
-            if (tokens.length >= 2) {
-                result = new String[2];
-                result[0] = tokens[tokens.length - 2];
-                result[1] = tokens[tokens.length - 1];
-            } else {
-                throw new IllegalArgumentException("Incorrect location value specified: " + location);
-            }
-        }
-        return result;
-    }
-
-    protected Condition buildCondition(String patientId, String fileName) throws Exception {
-        Condition condition = TestUtil.readLocalResource(fileName);
-        condition = condition.toBuilder().subject(Reference.builder().reference(string("Patient/" + patientId)).build()).build();
-
-        return condition;
-    }
-
-    /**
-     * Assert the the given {@link Bundle} is of the expected {@link BundleType} and has the expected entry count.
-     *
-     * @param bundle the bundle to tet
-     * @param expectedType
-     * @param expectedEntryCount
-     */
-    public static void assertResponseBundle(Bundle bundle, BundleType expectedType, int expectedEntryCount) {
-        assertNotNull(bundle);
-        assertNotNull(bundle.getType());
-        assertNotNull(bundle.getType().getValue());
-        assertEquals(expectedType.getValue(), bundle.getType().getValue());
-        if (expectedEntryCount > 0) {
-            assertNotNull(bundle.getEntry());
-            assertEquals(expectedEntryCount, bundle.getEntry().size());
-        }
-    }
-
-    /**
-     * Gets the self link of the bundle.
-     * @param bundle the bundle
-     * @return the self link, or null
-     */
-    protected String getSelfLink(Bundle bundle) {
-        for (Bundle.Link link : bundle.getLink()) {
-            String type = link.getRelation().getValue();
-            String uri = link.getUrl().getValue();
-            if ("self".equals(type)) {
-                return uri;
-            }
-        }
-        return null;
-    }
-}
+/*

```

Comment:
this is impossible to review, something isn't right about your eclipse formatting